src/pages/components/accordion/accessibility.mdx

```diff
@@ -7,423 +7,88 @@ description:
 tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 ---
 
-import {
-  StructuredListWrapper,
-  StructuredListHead,
-  StructuredListBody,
-  StructuredListRow,
-  StructuredListInput,
-  StructuredListCell,
-  OrderedList,
-  ListItem,
-} from 'carbon-components-react';
+<PageDescription>
+
+Design annotations are needed for specific instances shown below, but for the standard accordion component, Carbon already incorporates accessibility.
```

Comment:
Rephrase: It seemed off to me starting the page intro with a qualifier. 

> The standard Carbon accordion component already incorporates accessibility features, but further design annotations may be needed for specific instances shown below.

Reply:
@aagonzales The concern with that is if someone doesn't read the "but". It's more important that designers understand they _need_ to do annotations than that the standard one is hunky dory.

Reply:
I disagree, it makes it feel like the user needs to do a lot more work but we actually provide most of the work out of the box. Most of the "design annotations" so far also seem like optional updates so they don't _need_ to address them. I think the current phrasing gives too much emphasis on the wrong intention, things you have to do yourself vs things Carbon provides. Just my opinion.